ush/load_modules_run_task.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -l
```

Comment:
Does the bash script need to be login enabled in order for this work to function properly?  According to the code manager's guide, all bash scripts must explicitly be `#!/bin/bash` scripts (and not login enabled).

Reply:
Oh, good catch! I don't think that is necessary. I added it to see if I could figure out what was going on. I will remove it on my side and run a quick test to be sure.